### PR TITLE
docs: Explain HTTP based log level configuration

### PR DESCRIPTION
**Pull request by bripkens**
Dropwizard contains the task `LogConfigurationTask` and adds it by
default to the [admin environment](https://github.com/dropwizard/dropwizard/blob/49ff92712a936554d2cf9bd7a2b09a0fc07eaf25/dropwizard-core/src/main/java/io/dropwizard/setup/AdminEnvironment.java#L42). Unfortunately this task is not
yet documented.

### CODE REVIEW

File: docs/source/manual/core.rst [hunk 1]
```diff
@@ -804,6 +804,19 @@ appender with different configurations:
...
+Logging Configuration via HTTP
+------------------------------
+
+Active log levels can be changed during the runtime of a Dropwizard application via HTTP using
+the ``LogConfigurationTask``. For instance, to configure the log level for a
+single ``Logger``:
+
+.. code-block:: shell
+
+    curl -X POST -d "logger=org.stuartgunter.dropwizard&level=INFO" http://localhost:8081/tasks/log-level
```
Comment (jplock) [inline docs/source/manual/core.rst]: Would you mind changing the package here to be whatever would work out of the box from the `dropwizard-example` application?

Comment (bripkens) [inline docs/source/manual/core.rst]: Done

**Comment by bripkens** [comment]
@jplock: Changed the package name. Sorry for that!

**Comment by nickbabcock** [comment]
Looks good to me.

**Pull request merged**